gcexport.py

```diff
@@ -620,18 +687,18 @@ def extract_device(device_dict, details, start_time_seconds, args, http_caller,
 def load_gear(activity_id, args):
     """Retrieve the gear/equipment for an activity"""
     try:
-        gear_json = http_req(URL_GC_GEAR + activity_id)
+        gear_json = http_req_as_string(URL_GC_GEAR + activity_id)
         gear = json.loads(gear_json)
         if gear:
             del args # keep 'args' argument in case you need to uncomment write_to_file
-            # write_to_file(args.directory + '/activity_' + activity_id + '-gear.json',
+            # write_to_file(args.directory + sep + 'activity_' + activity_id + '-gear.json',
```

Comment:
```suggestion
            # write_to_file(join(args.directory, f'activity_{activity_id}-gear.json'),
```

Reply:
Python2 compatibility update
```suggestion
            # write_to_file(join(args.directory, 'activity_' + activity_id + '-gear.json'),
```